From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@ajftek__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

tysonn commented:
Reviewed and approved by article SME. Thanks!

adegeo committed:
Rewrite of 'how to deploy app'

genlin committed:
add new article

genlin committed:
change the file name

genlin committed:
correct typos

genlin committed:
correct the format issue

genlin committed:
correct format issue

genlin committed:
fix the format issue

genlin committed:
fix link issue

genlin committed:
update metadata

genlin committed:
correct typo

genlin committed:
update TOC

ktoliver committed:
Edit pass: active-directory-b2c-setup-sf-app-custom.md

Commit:
cabp02

ktoliver committed:
Update active-directory-b2c-advanced-audience-warning.md

burkeholland committed:
Update documentdb-nodejs-application.md

If possible, things should work cross-platform by default (as Node.js does). If they don't, let's save people a click and tell them that it's only on Windows.

**Note**: This change would cause us to have to keep this doc in sync when the local emulator for *nix systems ships.

burkeholland committed:
Update documentdb-nodejs-application.md

The wording was a little funny for me here.

Commit:
fixing broken links

Commit:
cabp02

Commit:
cabp02

Commit:
cabp02

genlin committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into master021

genlin committed:
apply edit changes

genlin committed:
update the date

genlin committed:
correct typo

genlin committed:
correct format issue

ealsur committed:
Adding Change Feed Processor SDK docs

ealsur committed:
Merge pull request #1 from MicrosoftDocs/master

Updating fork

ealsur committed:
Renaming and updating dropdown to include the new SDK

ealsur committed:
Updating dropdown to reflect Change Feed SDK

ealsur committed:
Updating dropdown to reflect Change Feed SDK

ealsur committed:
Updating dropdown to reflect Change Feed SDK

ealsur committed:
Updating dropdown to reflect Change Feed SDK

ealsur committed:
Updating dropdown to reflect Change Feed SDK

ealsur committed:
Updating TOC to reflect Change Feed SDK article

itechedit committed:
edit pass: active-directory-saas-kenexasurvey-tutorial

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into active-directory-saas-kenexasurvey-tutorial

itechedit committed:
edit pass: active-directory-saas-kenexasurvey-tutorial

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into active-directory-saas-kenexasurvey-tutorial

genlin committed:
apply tech comments

ealsur committed:
Fixing author

ealsur committed:
Fixing ms.author

itechedit committed:
edit pass: active-directory-saas-kenexasurvey-tutorial

itechedit committed:
edit pass: service-fabric-docker-compose

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into service-fabric-docker-compose

itechedit committed:
edit pass: service-fabric-docker-compose

atikmapari committed:
Bug : 1035118 - "PluralSight" replace the word "Pluralsight".

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into service-fabric-docker-compose

itechedit committed:
edit pass: service-fabric-docker-compose

itechedit committed:
edit pass: service-fabric-docker-compose

tfitzmac committed:
added consumer marketplace app

tfitzmac committed:
updated terms

itechedit committed:
edit pass: service-fabric-get-started-linux

v-stgreg committed:
Replaced obsolete and depricated properties with newer AzureStorageInfo method as required by Visual Studio 2017.

genlin committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into master021

genlin committed:
add editor changes

genlin committed:
update toc

Commit:
iot-hub updates

Commit:
Merge branch 'master' into patch-14

Commit:
Merge branch 'master' into patch-14

genlin committed:
correct typo

genlin committed:
add new file

genlin committed:
update toc

genlin committed:
update date

genlin committed:
apply edit changes

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into freshness

Commit:
freshness updates

Commit:
Update data-lake-analytics-manage-use-java-sdk.md

itechedit committed:
Update service-fabric-get-started-linux.md

CarlRabeler committed:
Update sql-database-service-tiers.md

CarlRabeler committed:
Update sql-database-service-tiers.md

Commit:
Update data-lake-analytics-manage-use-java-sdk.md

tamram committed:
Update batch-low-pri-vms.md

adegeo committed:
Refreshed mentions of 'preview' api

billmath committed:
updating freshness

PRMerger-2 committed:
Merge pull request #17151 from CarlRabeler/patch-298

PRMerger-2 committed:
Merge pull request #17144 from saveenr/patch-1

tfitzmac committed:
fix link

rjagiewich committed:
Merge pull request #17142 from Blackmist/freshness

Freshness

Lisaco88 committed:
Merge pull request #17085 from v-atmapa/master

Bug : 1035118 - "PluralSight" replace the word "Pluralsight".

Commit:
Retitled service descriptions article to state overview instead

tynevi committed:
deprecate comment guidance

ealsur committed:
Correcting gramar

PRMerger-2 committed:
Merge pull request #17153 from tamram/patch-2

Commit:
retiring FAQ and updating metadata

tynevi committed:
deprecating PR best practices

v-stgreg committed:
Updated to AzureStorage method as required by Visual Studio 2017

tfitzmac committed:
fixed link

Lisaco88 committed:
Merge pull request #17093 from tfitzmac/0710marketplace

added consumer marketplace app

tynevi committed:
Update create-images-markdown.md

Commit:
cabp02

tynevi committed:
Update create-links-markdown.md

ealsur committed:
Fixing the question mark position to be consistent

MarkMcGeeAtAquent committed:
Cosmos DB: fixed typ in redirect link

PRMerger-2 committed:
Merge pull request #17163 from v-stgreg/hdinsight

PRMerger-2 committed:
Merge pull request #17159 from tfitzmac/0712link

PRMerger-2 committed:
Merge pull request #17158 from billmath/aadconnect

billmath committed:
updating

Lisaco88 committed:
Merge pull request #16640 from MarkusVi/cabp02

cabp02

Commit:
Fixed broken link, typos in Bing APIs

Commit:
Updating Known Issue for PS

tfitzmac committed:
updated tags

itechedit committed:
Update service-fabric-docker-compose.md

Lisaco88 committed:
Merge pull request #17160 from jeffgilb/jeff712

Retitled service descriptions article to state overview instead

cychua committed:
Add build 578

tfitzmac committed:
fixed term

ShawnJackson committed:
Update active-directory-b2c-setup-sf-app-custom.md

cychua committed:
Update broken links

tynevi committed:
Update file-names-and-locations.md

ShawnJackson committed:
Merge pull request #16602 from ktoliver/active-directory-b2c-setup-sf-app-custom

Edit pass: active-directory-b2c-setup-sf-app-custom.md

ShawnJackson committed:
Update active-directory-saas-kenexasurvey-tutorial.md

genlin committed:
update toc

ShawnJackson committed:
Merge pull request #17042 from itechedit/active-directory-saas-kenexasurvey-tutorial

edit pass: active-directory-saas-kenexasurvey-tutorial

genlin committed:
fixes the issue

ronortloff committed:
Replicated table changes

removed reference to round robin.

PRMerger-2 committed:
Merge pull request #17173 from billmath/aadconnect

PRMerger-2 committed:
Merge pull request #16734 from burkeholland/patch-2

Commit:
Updating Known Issue for PS

ShawnJackson committed:
Merge pull request #17079 from itechedit/service-fabric-docker-compose

edit pass: service-fabric-docker-compose

Commit:
updates to test

GitHubber17 committed:
Merge pull request #17174 from swhite-msft/master

Fixed broken link, typos in Bing APIs

Commit:
Updating Known Issue for PS

ShawnJackson committed:
Merge pull request #17103 from itechedit/service-fabric-get-started-linux

edit pass: service-fabric-get-started-linux

Commit:
Updating Known Issue for PS

Commit:
Updating Known Issue for PS

SnehaGunda committed:
Adding login step

Ja-Dunn committed:
Merge pull request #16571 from genlin/master021

troubleshoot-device-names-problems.md

GitHubber17 committed:
Merge pull request #17176 from tfitzmac/0712tag

updated tags

PRMerger-2 committed:
Merge pull request #17181 from georgewallace/connectivityupdate

PRMerger-2 committed:
Merge pull request #17179 from ronortloff/master

PRMerger-2 committed:
Merge pull request #17177 from cychua/master

PRMerger-2 committed:
Merge pull request #16741 from burkeholland/patch-4

Ja-Dunn committed:
Merge pull request #16947 from ealsur/master

Adding Change Feed Processor SDK docs

GitHubber17 committed:
Merge pull request #17167 from MarkMcGeeAtAquent/patch-2

Cosmos DB: fixed typ in redirect link

GitHubber17 committed:
Merge pull request #17162 from HeathL17/hl-metadata

retiring FAQ and updating metadata

tynevi committed:
Update retire-or-rename-an-article.md

tynevi committed:
Update style-and-voice.md

GitHubber17 committed:
Merge pull request #17134 from genlin/master033

resource-manager-policy-requestdisallowedbypolicy-error.md

alkohli committed:
minor change

alkohli committed:
minor changes

tynevi committed:
Delete markdown-template-for-new-articles.md

Commit:
Merge pull request #17183 from SnehaGunda/postGA

Adding login step

Commit:
Merge pull request #17187 from alkohli/patch-115

minor change

Ja-Dunn committed:
Merge pull request #17175 from HeathL17/hl-relnotes

Updating Known Issue for PS

Commit:
Merge pull request #14136 from Thraka/fix-create

Fix VM Create sample

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

GitHubber17 committed:
Merge pull request #17124 from shizn/patch-14

iot-hub updates

Commit:
Merge pull request #17157 from Thraka/vmss-fix-api

Refreshed mentions of 'preview' api

v-alje committed:
Merge pull request #17198 from MicrosoftDocs/FromPublicRepo

From public repo

tynevi committed:
Merge pull request #17188 from MicrosoftDocs/tysonn-patch-1

Delete markdown-template-for-new-articles.md

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
